src/python3_pip_skeleton/__main__.py

```diff
@@ -106,6 +112,36 @@ def replace_in_file(file_path: Path, text_from: str, text_to: str):
             if child.suffix in CHANGE_SUFFIXES and child.name not in IGNORE_FILES:
                 text = replace_text(child.read_text())
                 child.write_text(text)
+            # Replace the file, ignoring line numbers
+            elif (
+                child.suffix in CHANGE_SUFFIXES
+                and child.name in IGNORE_FILES
+                and IGNORE_FILES[child.name]
+            ):
+                original_text = child.read_text()
+                replaced_text = replace_text(original_text)
+                for sub_strings in IGNORE_FILES[child.name]:
+                    assert isinstance(sub_strings, tuple)
+                    pre_sub_string, post_sub_string = sub_strings
+                    regex = rf"(?s){pre_sub_string}(.*?){post_sub_string}"
+                    # finditer so we can allow for different contents between ignore
+                    # substrings, e.g: `a foo b` and then later in the file `a bar b`
+                    original_substrings = list(re.finditer(regex, original_text))
```

Comment:
I think we should assert there is exactly one match, it's too easy to break otherwise